submissions/korotushak96/js-dom/index.css

```diff
@@ -0,0 +1,143 @@
+@media screen {
+    .burger {
```

Comment:
It seems to me, your burger menu switcher doesn't display properly. It is crooked a bit... 
Try to change `transform` properties for elements whit classes `.bar1, .bar2, .bar3`

Reply:
What do you mean? For my opinion it's good animation

Reply:
Try to compare [this](http://joxi.net/V2VO7odFx73xPA) and [this](http://joxi.net/J2bqzQDHXzZXnm)

Reply:
Sorry. I don't understand. Don't you like element's position or animation?

Reply:
Sorry. I can't understand you :(

Reply:
The problem is in translate properties in pixes:
`.change .bar1 {
        transform: rotate(-45deg) translate(-9px, 6px) ;
    }` and ` .change .bar3 {
        transform: rotate(45deg) translate(-8px, -8px) ;
    }`

Try to change them in order your close button will not crash!

Reply:
I'm not sure, I understand you, but I tried to fix it